src/RokuDeploy.ts

```diff
@@ -919,26 +923,74 @@ export class RokuDeploy {
         return outPkgFilePath;
     }
 
-    public async getDeviceInfo(options?: RokuDeployOptions) {
-        options = this.getOptions(options);
+    /**
+     * Get the `device-info` response from a Roku device
+     * @param host the host or IP address of the Roku
+     * @param port the port to use for the ECP request (defaults to 8060)
+     */
+    public async getDeviceInfo(options?: { enhance: true } & GetDeviceInfoOptions): Promise<DeviceInfo>;
+    public async getDeviceInfo(options?: GetDeviceInfoOptions): Promise<DeviceInfoRaw>
```

Comment:
Do you need a semicolon here?

Reply:
Yeah it's required. it's part of the syntax requirements for overloaded method signatures. 

https://www.tutorialsteacher.com/typescript/function-overloading